### PR TITLE
[FEATURE] admin sign in

### PR DESCRIPTION
**Pull request by IsaacSarte**
## Summary

- I added admin sign in feature

#### admin sign in route
image

#### bad request
image

#### admin signed in
image

### CODE REVIEW

**Review by Zhi0105 — APPROVED** [review]

File: src/components/Admin/Auth/Header.jsx [hunk 1]
```diff
@@ -0,0 +1,163 @@
...
+                                className="text-base font-medium text-gray-700 hover:text-gray-900 md:text-sm lg:text-xl">
+                                Admin
+                            </a>
+                            <a
+                                href="#contact"
+                                className="text-base font-medium text-gray-700 hover:text-gray-900 md:text-sm lg:text-xl">
+                                Contact Us
+                            </a>
+
+                        </Popover.Group> */}
```
Comment (analyn-cajocson) [inline src/components/Admin/Auth/Header.jsx]: Remove unnecessary comments

File: src/components/Admin/Auth/Header.jsx [hunk 2]
```diff
@@ -0,0 +1,163 @@
...
+                                            className="text-green-700 hover:text-green-600"
+                                        >
+                                            Sign in
+                                        </a>
+                                    </p>
+                                </div>
+                            </div>
+                        </div>
+                    </Popover.Panel>
+                </Transition> */}
```
Comment (analyn-cajocson) [inline src/components/Admin/Auth/Header.jsx]: Remove unnecessary comments

File: src/components/Admin/Auth/SignIn.jsx [hunk 3]
```diff
@@ -0,0 +1,139 @@
...
+            .post(url, {
+                admin: {
+                    email: email,
+                    password: password
+                }
+            })
+            .then((res) => handleHeader(res))
+            .catch((e) => {
+                setError(e.response.data.error);
+                console.log(e.response.data.error);
```
Comment (analyn-cajocson) [inline src/components/Admin/Auth/SignIn.jsx]: Remove console.log

File: src/services/AdminAuthAPI.js [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+      localStorage.removeItem("adminAuth");
+    };
+  
+    return {
+      isAuthenticated,
+      logout,
+    };
+  };
+  
+  export default AdminAuthAPI();
```
Comment (analyn-cajocson) [inline src/services/AdminAuthAPI.js]: Fix code alignment

**Review by analyn-cajocson — CHANGES_REQUESTED** [review]

**Review by analyn-cajocson — APPROVED** [review]

**Pull request merged**